docs/geps/gep-04.rst

```diff
@@ -186,7 +186,35 @@ This is not satisfactory; ideally, functions would only be present in years wher
     - Users putting together their policy environment based on functions from different years
     - ...
 
+Additional functionalities
+--------------------------
 
+We implemented a small set of additional features that simplify the specification of
+certain types of functions of the tax and transfer system.
+
+Aggregation of variables
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Arguments of functions can also be aggregations of input variables or outputs of other
+functions on the tax unit or household level. In the example above, one argument of the
+function ``soli_st_tu`` is ``abgelt_st_tu``. This variable would be typically specified
+as a function on the tax unit level (``abgelt_st_tu``) as it is done above. However, if
+a function with this name does not exist, but a function on the individual level with
+the same name excluding the ``_tu`` suffix (``abgelt_st``), the output of this
+individual level function is just summed up on the tax unit level before it enters the
+function ``soli_st_tu``.
+
+This feature avoids the need to define a bunch of similarly looking functions that only
+aggregate values on certain levels. Similarly, functions with the ``_hh`` can be
+aggregations on the household level.
```

Comment:
Just refer to all units defined in GEP-01?

As a general note on this section, I think I would prefer the motivation in the last paragraph first and describe the concept in an abstract faction. Then an example.

Reply:
I restructured the section as suggested.